src/v/raft/group_manager.cc

```diff
@@ -203,4 +223,19 @@ void group_manager::setup_metrics() {
         sm::description("Number of raft groups"))});
 }
 
+ss::future<> group_manager::flush_groups() {
```

Comment:
nit: better note somewhere that the method assumes that the gate is already held.